File: app/assets/javascripts/red_base/dashboard/models/permission.js
Dashboard.Permission = DS.Model.extend(Dashboard.DModel, {
    name: DS.attr('string'),
    string: DS.attr('string'),
    groups: DS.hasMany("group")
});

Dashboard.Permission.FIXTURES =[
    {id: 1, "name":"read|RedBase::User", "string":"can read Red base/user"},
    {id: 2, "name":"update|RedBase::User","string":"can update Red base/user"}
];

File: app/models/red_base/permission.rb
module RedBase
  class Permission < ActiveRecord::Base

    has_and_belongs_to_many :groups

    def string_repr
      _("can %s %s") % [_(self.permission_type.to_s), self.model.underscore.humanize]
    end

    def id_repr
      "#{self.permission_type.to_s}|#{self.model}"
    end

  end
end
